tests/flow/test_bruteforce.py

```diff
@@ -3,6 +3,79 @@
 # the Server Side Public License v1 (SSPLv1).
 
 from common import *
+def test_bytearray():
```

Comment:
Consider having more meaningful name, or adding a comment describing what we are testing here